src/main/kotlin/device/IAudioRecorder.kt

```diff
@@ -0,0 +1,9 @@
+package device
+
+import io.reactivex.Observable
+
+interface IAudioRecorder {
+    fun record()
```

Comment:
how about start()

Reply:
Not currently. Is that something we should add?